app/models/document.rb

```diff
@@ -239,7 +239,7 @@ def publish!
         email_alert_api.send_alert(EmailAlertPresenter.new(self).to_json)
       end
 
-      publish_request.code == 200 && rummager_request.code == 200
+      publish_request.code == 200 && rummager_request.code == 202
```

Comment:
Note that rummager has returned `202` in production for years (https://github.com/alphagov/rummager/pull/124 - July 2013). I've removed the special development behaviour recently, because I suspected it could cause problems for people using and testing things in dev (https://github.com/alphagov/rummager/pull/629). 

Seeing as above code would've suddenly and mysteriously failed in production (because `rummager_request.code == 200`), I'm happy about such a quick return on investment :)

As for the questions: I think we haven't properly discussed how to give feedback to users about the state of the publishing pipeline.

In an ideal world I would like to see every content item in publisher apps have a status indicator 🚦 for all async side-effects:

- published to content-store
- indexed in search
- email alerts sent